web-client/integration-tests/journey/docketClerkAddsTranscriptDocketEntryFromOrder.js

```diff
@@ -55,10 +56,18 @@ export const docketClerkAddsTranscriptDocketEntryFromOrder = (
       value: 'Anything',
     });
 
+    // suppressing moment deprecation warnings temporarily as moment returns an
+    // invalid object when called with a single day or month value
+    moment.suppressDeprecationWarnings = true;
```

Comment:
Why couldn't we use a two-digit value in the test instead of removing the warning?

Reply:
Agreed, to keep things clean, frankly I would prefer that no other tests or files ever need know that `moment` even exists.

Reply:
The `updateCourtIssuedDocketEntryFormValueSequence` only updates on form value at a time, and regenerates the document title each time it is called. Calling the sequence with only one day/month value results in a `undefined-month-undefined` or `day-undefined-undefined` string being passed into moment.

Reply:
The timestamp business is a can of worms, and this function is in the middle of it.  The mmmm-yy-dd format that's being constructed/deconstructed is really more of a presenter-related validation function IMO and doesn't belong in DateHandler IMO.  If it were pulled out, it would also solve the issue of moment ever seeing `undefined`.

Reply:
So does this seem maybe like an opportunity for refactoring in the sequence? Not 100% sure what that looks like, but I'd prefer that over disabling a deprecation warning if we can help it.